app/views/comments/_form.html.erb

```diff
@@ -51,7 +51,7 @@
               <!-- http://stackoverflow.com/questions/11235206/twitter-bootstrap-form-file-element-upload-button -->
               <label id="input_label" class="" for="fileinput">
                 <input id="fileinput" type="file" name="image[photo]" style="display:none;" />
-                <a onClick="handleClick()" class="d-none d-md-inline">choose one</a>
+                <a onClick="handleClick()" class="d-none d-md-inline" style="text-decoration:underline;">choose one</a>
```

Comment:
Could you make this a css class?

Reply:
I don't see any CSS class that only add a `text-decoration: underline`.

Do you want me to add such a CSS class (that could be used on others elements) ?
```html
<a onClick="handleClick()" class="d-none d-md-inline text-underline">choose one</a>
```
```css
.text-underline {
    text-decoration: underline;
}
```

Or simply add a `text-decoration: underline` CSS property on this specific link ?

Reply:
Totally up to you! It could be helpful so have such classes if it's used elsewhere. Personally i feel like the style tag is a bit of an anti-pattern but has it's place.

Reply:
Just added a CSS class